hint.cabal

```diff
@@ -15,7 +15,7 @@ license:      BSD3
 license-file: LICENSE
 author:       The Hint Authors
 maintainer:   mvdan@mvdan.cc
```

Comment:
If anyone has any ideas on what to do with this, please speak up. I don't think my email should be here - at least not on its own.

Is it possible to say "a changing group of people", or to not give an email address? Without going as far as saying that noone is maintaining the package.